doc/nrf/includes/boardname_tables/sample_boardnames.txt

```diff
@@ -1,7 +1,6 @@
 .. _sampleboardnames:
 
-Set used by BLE (Central BAS, Central DFU SMP, Central HIDS, Central UART, EnOcean, Mesh Light, Mesh Light Switch, Peripheral HIDS Keyboard,  Peripheral HIDS mouse,
-Peripheral UART, NUS Shell Transport, Throughput) and NFC samples
+Set used by sample (Radio Test)
```

Comment:
Nice, there are atleast 3 open PRs that touch this line, I hope this is the one that will stand